app/src/main/java/com/superproductivity/superproductivity/CommonJavaScriptInterface.kt

```diff
@@ -439,13 +499,17 @@ abstract class CommonJavaScriptInterface(
             Log.d("SuperProductivity", "Before SimpleStorageHelper callback func def")
             // Register a callback with SimpleStorage when a folder is picked
             activity.storageHelper.onFolderSelected =
-                { _, root -> // could also use simpleStorageHelper.onStorageAccessGranted()
+                { requestCode, root -> // could also use simpleStorageHelper.onStorageAccessGranted()
                     Log.d("SuperProductivity", "Success Folder Pick! Now saving...")
                     // Get absolute path to folder
                     val fpath = root.getAbsolutePath(activity)
                     // Open preferences to save folder to path
                     val sp = activity.getPreferences(Context.MODE_PRIVATE)
                     sp.edit().putString("filesyncFolder", fpath).apply()
+                    // Once permissions are granted, callback web application to continue execution
+                    callJavaScriptFunction(
+                        FN_PREFIX + "grantFilePermissionCallBack('" + requestCode + "')"
```

Comment:
```suggestion
                        FN_PREFIX + "grantFilePermissionCallBack('" + requestId + "')"
```

Reply:
Thank you, missed this typo caused by a copy/paste, it's now merged!